console/views.py

```diff
@@ -20,9 +22,13 @@ def home():
     return "stuff"
```

Comment:
*I'm just pointing this out - don't worry about changing it as it's not part of this pull request +1